doc/installation.rst

```diff
@@ -85,57 +94,73 @@ On macOS, the first time you open Spyder, you may see a message that it cannot b
+To run the bundled version of Spyder after installing it with Anaconda, the recommended method on Windows is to launch it via the Start menu shortcut.
 On other platforms, open Anaconda Navigator, scroll to Spyder under ``Home``, and click ``Launch``.
 
 .. image:: /images/installation/installation-anaconda-navigator.png
    :alt: Anaconda Navigator showing Spyder
 
 If Spyder does not launch via this method or you prefer to use the command line, open Anaconda Prompt (Windows) or your terminal (other platforms), type ``conda activate base`` then ``spyder``.
 
+.. note::
 
-New Conda environment
-~~~~~~~~~~~~~~~~~~~~~
+   The bundled Spyder version can often be quite out of date, missing new features and bug fixes from the latest version, and if you install, change or remove other packages, there is a chance of dependency conflicts or a broken Spyder installation.
+   We recommend :ref:`installing Spyder into a new Conda environment <installation-miniconda>` to avoid all these issues.
 
-If you would like to have Spyder in a dedicated environment to update it separately from your other packages and avoid any conflicts, you can.
-Just run the following command in your Anaconda Prompt (Windows) or terminal (other platforms), for a minimal install of Spyder into a new environment called ``spyder-env``:
 
-.. code-block:: bash
+.. _installation-miniconda:
 
-   conda create -n spyder-env spyder
+Installing with Miniconda
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To install Spyder's optional dependencies as well for full functionality, use the following command instead:
+Once you download and install Miniconda, or to get a more reliable and up-to-date Spyder install with Anaconda, install Spyder into its own fresh environment.
+For a full install of Spyder and all optional dependencies, run the following command in your Anaconda Prompt (Windows) or terminal (other platforms):
 
-.. code-block:: bash
+.. code-block:: shell
+
+   conda create -c conda-forge -n spyder-env spyder numpy scipy pandas matplotlib sympy cython
+
+For a minimal install without the optional functionality and integration with the above packages, you can instead run:
 
-   conda create -n spyder-env spyder numpy scipy pandas matplotlib sympy cython
+.. code-block:: shell
 
-To install Spyder from Conda-Forge instead of the default Anaconda channel, add ``-c conda-forge`` at the end of either of the previous commands.
+   conda create -c conda-forge -n spyder-env spyder
 
-You can then run Spyder by the same methods as above, except make sure to select the start menu shortcut with ``(spyder-env)`` in the name, select the ``spyder-env`` environment on the left before launching it with Navigator, or type ``conda activate spyder-env`` before launching it on the command line.
+This installs Spyder into a new environment called ``spyder-env``, using the more up-to-date, community Conda-Forge channel.
```

Comment:
```suggestion
This will install Spyder into a new environment called ``spyder-env``, using the more up-to-date, community Conda-Forge channel.
```

I'm not sure if this correct. If not, please dismiss it.

Reply:
Either is perfectly valid, but "installs" is a bit more concise and direct, so I didn't end up applying this.